### ISSUE TITLE
OOC Bug

### ISSUE THREAD

**Issue by 1avx1** [vingard/impulse-issues#90]
**Describe the bug**
You can send OOC messages even if you have OOC disabled

**To reproduce**
Steps to reproduce the behaviour:
1. Bind a key to "/ooc *text*"
Example: bind k "say /ooc when the ied is sus!"
2. Disable OOC from F1 - Settings - Chatbox
3. Press "k"
4. See error (push notification and the message in console)

**Expected behaviour**
It shouldn't happen at all.

**Screenshots**
https://i.imgur.com/4x6UzyV.png

**impulse version (press F1 in the bottom right):**
1.61
**Lua error**
No LUA errors

**Additional context**
Discovered by accident lol

**Comment by NICKOLAS78GR** [comment]
Yeah but, I like that bug, it lets me write in OOC without having my chat filled with "amogusfunni" when I try to RP, also it doesn't affect anyone.
